streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1871,12 +1871,12 @@ boolean hasSubscriptionUpdates() {
 
     synchronized void updateSubscribedTopics(final Set<String> topics,
                                              final String logPrefix) {
-        log.debug("{}found {} topics possibly matching subscription", logPrefix, topics);
+        log.debug("{}found {} topics possibly matching subscription", logPrefix, topics.size());
```

Comment:
Thanks for the PR! I just noticed these two log messages are completely redundant; can we remove one of them, or combine into a single log?